transport/http/src/main/java/org/kaazing/gateway/transport/http/bridge/filter/HttpProtocolCompatibilityFilter.java

```diff
@@ -425,6 +417,19 @@ else if (!PROTOCOL_HTTPXE_1_1.equals(nextProtocol)) {
             }
         }
 
+        // Use .ksn for x-sequence-no header for wse/1.0, httpxe/1.1 cases
+        // other cases, don't do anything as some services (for e.g http.proxy) want to pass through
+        String protocol = httpRequest.getHeader(HEADER_X_NEXT_PROTOCOL);
```

Comment:
No, the code above is setting the HEADER_X_NEXT_PROTOCOL to PROTOCOL_HTTPXE_1_1 and to PROTOCOL_WSE_1_0. So, that may not work.